pages/jdl/deployments.md

```diff
@@ -50,14 +50,14 @@ deployment {
 If you want more than one deployment, here's how you do it:
 
 ```
-// will be created under 'docker-compose' folder
+// will be created under the 'docker-compose' folder
 deployment {
   deploymentType docker-compose
   appsFolders [foo, bar]
   dockerRepositoryName "yourDockerLoginName"
 }
 
-// will be created under 'kubernetes' folder
+// will be created under the 'kubernetes' folder
 deployment {
   deploymentType kubernetes
```

Comment:
Just curious - is it possible to change the name of the folder that's generated? For example, `k8s` instead of `kubernetes`?

Reply:
I don't know if there's an option for that, but it isn't possible (from what I've seen) with the JDL, yet.